fluentd-gcp-image/Makefile

```diff
@@ -18,11 +18,18 @@
 .PHONY:	build push
 
 PREFIX=staging-k8s.gcr.io
-TAG = 2.0.19
+TAG = 2.1.0
+BUILD_DEPS="make gcc g++ libc6-dev ruby-dev libffi-dev"
 
 build:
 	docker build --pull -t $(PREFIX)/fluentd-gcp:$(TAG) .
 
 
 push:
 	gcloud docker -- push $(PREFIX)/fluentd-gcp:$(TAG)
+
+
+refreeze-dependencies: build
```

Comment:
Why not just "update-dependencies"?

Reply:
I use this work to emphasize that it will not only update Gemfile.lock but also remove version boundaries for all packages and select newest available.

Reply:
This is what I would expect the word update to mean. Can you clarify in README what this command does, exactly?

Reply:
I decided to rename to "update-dependencies" to match common nomenclature.